src/lib/OpenEXRCore/internal_coding.h

```diff
@@ -11,19 +11,6 @@
 
 #include "internal_structs.h"
 
-#include <ImathConfig.h>
```

Comment:
what is wrong with this dependency? maybe instead we use __has_include?

Reply:
For a first pass at the single file concept, I simply sliced out the Imath dependency, as we're only using it for half conversions. I'll give this another think.